### PR TITLE
Form control> Readonly plain text: Fix font weight of email address

### PR DESCRIPTION
**Pull request by isabellechanclou**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_
 
 ### Related issues
 
 #1813
 
 ### Description
 
 The email address in readonly plain text should be in bold and not in regular font weight.
 
 ### Motivation & Context
 
 Fix asked by design when working on the design callout messages (avoids one)/
 
 ### Types of change
 
 - Bug fix (non-breaking which fixes an issue)
 
 ### Live previews
 
 - <https://deploy-preview-2006--boosted.netlify.app/docs/5.3/forms/form-control/#readonly-plain-text>
 
 ### Checklist
 
 
 
 
 #### Contribution
 
 - [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)
 
 #### Accessibility
 
 - [x] My change follows accessibility good practices; I have at least run axe
 
 #### Design
 
 - [x] My change respects the design guidelines defined in [Orange Design System](https://system.design.orange.com/0c1af118d/p/8118d1-web)
 - [x] My change is compatible with responsive display
 
 #### Development
 
 - [x] My change follows the [developer guide](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/wiki/Developer-guide)
 - [ ] I have added JavaScript unit tests to cover my changes
 - [ ] I have added SCSS unit tests to cover my changes
 
 #### Documentation
 
 - [x] My change introduces changes to the documentation and/or I have updated the documentation accordingly
 
 ### Checklist (for Core Team only)
 
 - [ ] My change introduces changes to the migration guide
 - [ ] My new component is added in Storybook
 - [ ] My new component is compatible with RTL
 - [ ] Manually run [BrowserStack tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/browserstack.yml)
 - [ ] Manually test browser compatibility with BrowserStack (Chrome >= 60, Firefox >= 60 (+ ESR), Edge, Safari >= 12, iOS Safari, Chrome & Firefox on Android)
 - [x] Code review
 - [x] Design review
 - [ ] A11y review
 
 #### After the merge
 
 - [ ] Manually launch [Percy tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/percy.yml)

### CODE REVIEW

File: scss/forms/_form-control.scss [hunk 1]
```diff
@@ -131,6 +131,7 @@
   width: 100%;
   padding: $input-padding-y 0;
   margin-bottom: 0; // match inputs if this class comes on inputs with default margins
+  font-weight: $input-font-weight; //boosted mod
```
Comment (MewenLeHo) [inline scss/forms/_form-control.scss:134]:
```suggestion
  font-weight: $input-font-weight; // Boosted mod
```

**Review by MewenLeHo — COMMENTED** [review]
Looks good.

But `.form-control-plaintext` is used to make text appear like regular plain text. And regular plain text isn't supposed to be bold. It may be worth it to update the doc and the CSS comment in the CSS file (from line 124 to 127).

**Comment by MewenLeHo** [comment]
Any need to update the migration guide?

**Comment by julien-deramond** [comment]
Just a question here, are we sure that we can distinguish enough what are the labels and the values when we have a form with a vertical layout?

Screenshot 2023-05-12 at 09 35 11

A lot of stuff in bold on a page for a big form. Don't have a strong opinion on it but I prefer to raise the question.
With a horizontal layout, reading is less of an issue because the elements are more "separated" (left for the labels, right for the values).

**Comment by isabellechanclou** [comment]
This question has been risen in web spec meeting. The design is in charge and will come back to us when ready.

**Pull request closed without merging**